WebApp/src/app/calendar-overview/calendar/calendar.service.ts

```diff
@@ -129,12 +132,21 @@ export class CalendarService {
     });
   }
 
-  findFetchedCalendarItems(month: Date): any {
+  findSurroundingFetchedCalendarItems(month: Date): CalendarItem[] {
```

Comment:
findSurroundingFetchedCalendarItems

Je hebt ook al calendarItems die niet gefetched zijn?
Naamgeving maakt de code nogal lastig te begrijpen.

Reply:
Met surrounding bedoel ik de maanden die voor en na de huidige maand vallen. Wanneer er maanden gefetched worden worden deze opgeslagen. Er wordt iedere keer gekeken of een maand en/of de aangrenzende maanden al gefetched zijn. Zo niet, wordt de nog niet gefetchde maand opgehaald. _FoundCalendarItems_ is dus een collectie van _foundCalendarItemsThisMonth_ en _foundSurroundingCalendarItems_. Om het duidelijker te maken heb ik _fetched_ naar _stored_ vernoemd.